api-samples/alarms/index.js

```diff
@@ -94,17 +93,12 @@ class AlarmManager {
   };
 
   async cancelAlarm(name) {
-    // TODO: Remove custom promise wrapper once the Alarms API supports promises
-    return new Promise((resolve) => {
-      chrome.alarms.clear(name, (wasCleared) => {
-        if (wasCleared) {
-          this.logMessage(`Manager: canceled alarm "${name}"`);
-        } else {
-          this.logMessage(`Manager: could not canceled alarm "${name}"`);
-        }
-
-        resolve(wasCleared);
-      });
+    await chrome.alarms.clear(name, (wasCleared) => {
```

Comment:
Why await here and not a return? I think it'd be better to leave the decision whether the promise should be awaited to the caller.